### PR TITLE
check method before capturing trace.

### PR DESCRIPTION
**Pull request by vrabeshko**
revert jest 24 version

### CODE REVIEW

**Comment by vrabeshko** [comment]
[JS SDK errors](https://trello.com/c/YLFtcU4L/1881-js-sdk-errors)

File: src/xcoobee/sdk/System.js [hunk 1]
```diff
@@ -407,17 +395,13 @@ class System {
...
-          payload,
-          pgpSecret,
-          pgpPassword
-        );
-
-        event.payload = JSON.parse(encryptedPayload);
-      } catch (e) {
-        // Do nothing, we will pass on the payload as it is.
-      }
+      const payloadJson = await decryptWithEncryptedPrivateKey(
```
[the remark this answers is not in the document]
Comment (vrabeshko) [reply src/xcoobee/sdk/System.js:398]: `payload` already declared in upper scope

File: src/xcoobee/core/XcooBeeError.js [hunk 2]
```diff
@@ -11,7 +11,11 @@ class XcooBeeError extends Error {
   constructor(...args) {
     // See https://medium.com/@xjamundx/custom-javascript-errors-in-es6-aa891b173f87
     super(...args);
-    Error.captureStackTrace(this, XcooBeeError);
+    if (Error.captureStackTrace) {
+      Error.captureStackTrace(this, XcooBeeError);
+    } else {
+      this.stack = (new Error(...args)).stack;
```
[the remark this answers is not in the document]
Comment (vrabeshko) [reply src/xcoobee/core/XcooBeeError.js:17]: captureStackTrace is not available in firefox, so we emulate same behaviour here

File: package.json [hunk 3]
```diff
@@ -87,8 +86,8 @@
     "eslint-config-airbnb-base": "^14.1.0",
     "eslint-plugin-import": "^2.13.0",
     "husky": "^4.2.5",
-    "jest": "^25.3.0",
-    "jest-config": "^25.3.0",
+    "jest": "^24.8.0",
+    "jest-config": "^24.8.0",
```
[the remark this answers is not in the document]
Comment (vrabeshko) [reply package.json]: updated to 26.1 and fix tests

[the remark this answers is not in the document]
Comment (Bilal-S) [reply package.json]: @dhurlburtusa can we merge or are we waiting on other items?

File: README.md [hunk 4]
```diff
@@ -53,13 +55,14 @@ npm run build
...
+Don't forget that if you need to use the openpgp to decrypt payloads you need to load the OpenPGP library files as well. Either via the openpgp CDN or from your own site. You can use cdnjs via `https://cdnjs.com/libraries/openpgp` or our `https://app.xcoobee.net/scripts/sdk`.
+
+`<script src='https://app.xcoobee.net/scripts/sdk/openpgp.min.js' />`
 
 Example of load with openpgp library all from your own source:
 
 ```
 <script src='openpgp.min.js' />
-<script src='openpgp.worker.js' />
 <script src='xcoobee-sdk-0.9.5.web.js' />
```
[the remark this answers is not in the document]
Comment (Bilal-S) [reply README.md]: Latest version published is `1.4.0`

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (dhurlburtusa) [reply package.json]: @Bilal-S We can merge now.

**Pull request merged**